tests.py

```diff
@@ -0,0 +1,303 @@
+from os.path import join
+import pytest
+
+from pyminio import PyminioTest
+
+ROOT = '/'
+FILE_CONTENT = bytes('test', 'UTF-8')
```

Comment:
If I understand correctly, this will work as well:
```suggestion
FILE_CONTENT = b'test'
```